docs/core/policies.rst

```diff
@@ -197,56 +197,59 @@ set the ``random_seed`` attribute of the ``KerasPolicy`` to any integer.
 Embedding Policy
 ^^^^^^^^^^^^^^^^
 
-Transformer Embedding Dialogue Policy (TEDP)
+    .. warning::
+
+        ``EmbeddingPolicy`` got renamed to ``TEDPolicy``. Please use :ref:`ted_policy` instead of ``EmbeddingPolicy``.
+        The functionality of the policy stayed the same.
+
+
+.. _ted_policy:
 
-Transformer version of the Recurrent Embedding Dialogue Policy (REDP)
-used in our paper: `<https://arxiv.org/abs/1811.11707>`_
+TED Policy
+^^^^^^^^^^
+
+Transformer Embedding Dialogue (TED) Policy
+
+The policy used in our paper https://arxiv.org/abs/1910.00486.
 
 This policy has a pre-defined architecture, which comprises the
 following steps:
 
-    - concatenate user input (user intent and entities),
-      previous system action, slots and active form
-      for each time step into an input vector
-      to pre-transformer embedding layer;
+    - concatenate user input (user intent and entities), previous system action, slots and active form for each time
+      step into an input vector to pre-transformer embedding layer;
     - feed it to transformer;
-    - apply a dense layer to the output of the transformer
-      to get embeddings of a dialogue for each time step;
+    - apply a dense layer to the output of the transformer to get embeddings of a dialogue for each time step;
     - apply a dense layer to create embeddings for system actions for each time step;
-    - calculate the similarity between the
-      dialogue embedding and embedded system actions.
-      This step is based on the
-      `StarSpace <https://arxiv.org/abs/1709.03856>`_ idea.
+    - calculate the similarity between the dialogue embedding and embedded system actions.
+      This step is based on the `StarSpace <https://arxiv.org/abs/1709.03856>`_ idea.
 
-It is recommended to use
-``state_featurizer=LabelTokenizerSingleStateFeaturizer(...)``
+It is recommended to use ``state_featurizer=LabelTokenizerSingleStateFeaturizer(...)``
 (see :ref:`featurization_conversations` for details).
 
 **Configuration:**
 
     Configuration parameters can be passed as parameters to the
-    ``EmbeddingPolicy`` within the policy configuration file.
+    ``TEDPolicy`` within the configuration file.
 
     .. warning::
 
-        Pass an appropriate number of ``epochs`` to the ``EmbeddingPolicy``,
-        otherwise the policy will be trained only for ``1``
-        epoch.
+        Pass an appropriate number of ``epochs`` to the ``TEDPolicy``, otherwise the policy will be trained only
```

Comment:
what is an "appropriate number"?

Reply:
will be part of a separate PR https://github.com/RasaHQ/rasa/issues/5308